problems/prob3.rb

```diff
@@ -0,0 +1,24 @@
+#Given an array of numbers having size N in which every number is between 1 and N,
+#determine if there are any duplicates in it. You are allowed to destroy the array if you like.
+#Do not sort the array or use bit vectors. Try to work within the array without using any other temporary
+#data structures. Do not use 2 nesting for loops since the time complexity would be high.
+#Try to find a solution by executing just one loop.
+#Hint: manipulate the elements of the same array as you loop through.
+
+def check_duplicates(numbers)
```

Comment:
If you payed attention to `#nil?` and `#empty?` methods, they have `?` at end of the name, meaning the methods return `boolean`. This is a ruby convention for such methods. Could you use that too?